src/sentry/static/sentry/app/views/projectReleaseTracking.jsx

```diff
@@ -0,0 +1,252 @@
+    return {
+      loading: true,
+      error: false,
+      pluginList: [],
+      webhookUrl: '',
+      token: ''
+    };
+  },
+
+  componentDidMount() {
+    this.fetchData();
+  },
+
+  fetchData() {
+    let {orgId, projectId} = this.props.params;
+    let done = _.after(2, () => {
+      this.setState({loading: false});
+    });
+    this.api.request(`/projects/${orgId}/${projectId}/releases/token/`, {
+      method: 'GET',
+      success: data =>
+        this.setState({
+          webhookUrl: data.webhookUrl,
+          token: data.token
+        }),
+      error: () => {
+        this.setState({
+          error: true
+        });
+      },
+      complete: done
+    });
+    this.api.request(`/projects/${orgId}/${projectId}/plugins/`, {
+      success: data => {
+        this.setState({
+          pluginList: data.filter(p => p.type === 'release-tracking')
+        });
+      },
+      error: () => {
+        this.setState({
+          error: true
+        });
+      },
+      complete: done
+    });
+  },
+
+  onEnablePlugin(plugin) {
+    this.setState({
+      pluginList: this.state.pluginList.map(p => {
+        if (p.id !== plugin.id) return p;
+        return {
+          ...plugin,
+          enabled: true
+        };
+      })
+    });
+  },
+
+  onDisablePlugin(plugin) {
+    this.setState({
+      pluginList: this.state.pluginList.map(p => {
+        if (p.id !== plugin.id) return p;
+        return {
+          ...plugin,
+          enabled: false
+        };
+      })
+    });
+  },
+
+  onSubmit(evt) {
+    evt.preventDefault();
+    this.regenerateToken();
+  },
+
+  regenerateToken() {
+    let {orgId, projectId} = this.props.params;
+    this.api.request(`/projects/${orgId}/${projectId}/releases/token/`, {
+      method: 'POST',
+      data: {project: projectId},
+      success: data =>
+        this.setState({
+          token: data.token,
+          webhookUrl: data.webhookUrl
+        }),
+      error: () => {
+        this.setState({
+          error: true
+        });
+      }
+    });
+  },
+
+  getReleaseWebhookIntructions() {
+    let webhookUrl = this.state.webhookUrl;
+    return (
+      'curl ' +
+      webhookUrl +
+      ' \\' +
+      '\n  ' +
+      '-X POST \\' +
+      '\n  ' +
+      "-H 'Content-Type: application/json' \\" +
+      '\n  ' +
+      '-d \'{"version": "abcdefg"}\''
+    );
+  },
+
+  getReleaseClientConfigurationIntructions() {
+    return (
+      '// See SDK documentation for language specific usage.' +
+      '\n' +
+      "Raven.config('your dsn', {" +
+      '\n' +
+      '  ' +
+      "release: '0e4fdef81448dcfa0e16ecc4433ff3997aa53572'" +
+      '\n' +
+      '});'
+    );
+  },
+
+  render() {
+    let {organization, project} = this.props;
+    let {pluginList} = this.state;
+    if (this.state.loading) return <div className="box"><LoadingIndicator /></div>;
+    else if (this.state.error) return <LoadingError onRetry={this.fetchData} />;
+
+    return (
+      <div>
```

Comment:
i think we want to also check project permissions here with `access.has('project:write')` (unless we are doing that in a parent react view already, which is possible)